docs/core/porting/third-party-deps.md

```diff
@@ -63,31 +61,53 @@ dotnet5.4
 dotnet5.5
 ```
 
-**While these will likely work with your code, there is no guarantee of compatibility**.  Packages with these TFMs were built with pre-release .NET Core packages.  Take note of when (or if) packages like this are updated to be `netstandard`-based.
+**While these TFMs likely work with your code, there is no guarantee of compatibility**. Packages with these TFMs were built with pre-release .NET Core packages. Take note of when (or if) packages using these TFMs are updated to be .NET Standard-based.
 
 > [!NOTE]
-> To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `imports` directive in your `project.json` file.
+> To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `PackageTargetFallback` MSBuild element in your project file.
```

Comment:
How do I do that? Would a link to some documentation about this (possibly https://docs.microsoft.com/nuget/schema/msbuild-targets#packagetargetfallback) be useful here?

Reply:
That's a good suggestion @svick, while we still don't have our own MSBuild reference topic for .NET Core.